circle.yml

```diff
@@ -301,6 +301,9 @@ commands:
       - run:
           name: Install Node Modules
           command: |
+            # avoid installing Percy's Chromium every time we use @percy/cli
```

Comment:
This is already merged in develop (https://github.com/cypress-io/cypress/pull/21218), but adding it here since the develop -> 10.0-release merge is a bit delayed